src/typed_ast.hpp

```diff
@@ -317,4 +317,14 @@ struct MonoTypeHandle : public TypedAST {
 	    : TypedAST {TypedASTTag::MonoTypeHandle} {}
 };
 
+struct Constructor : public TypedAST {
+	MonoTypeHandle* m_mono;
```

Comment:
Why not use a `MonoId` instead of a node here?

Reply:
Also, this `m_mono` probably be defaulted to `nullptr`, just in case

Reply:
Good point, I can't remember what lead me to store a handle, but all the info I need requires only the id.